docs/UserGuide.md

```diff
@@ -83,16 +83,16 @@ Adds a new entry to their address book for financial advisors, including persona
 
 #### Example commands:
 * `add n/John Doe p/98765432 e/johnd@example.com o/Barber a/John Street, Block 123, #01-01 appt/12-12-2023 08:30`
-* `add n/Betsy Crowe t/Friend e/betsycrowe@example.com o/Entrepreneur a/Newgate Prison p/1234567 t/Criminal`
+* `add n/Betsy Crowe t/Friend e/betsycrowe@example.com o/Entrepreneur a/Newgate Prison p/1234567 t/Friend`
```

Comment:
Almost perfect! I think can remove the Friend tag at the beginning since currently there are two Friend tags.